econplayground/main/migrations/0082_auto_20230202_1102.py

```diff
@@ -1,56 +0,0 @@
-# Generated by Django 3.2.17 on 2023-02-02 16:02
```

Comment:
Is there a reason you've deleted these migration files? These migrations have already been run, so removing the files can cause weird issues. We generally keep all migration files in the source repository, except in really special cases.

Reply:
Understood. Locally I was deleting them because I thought someone told me to keep db migrations to one per PR. I can restore the ones that have already been merged.